services/object/test/backends/google_test.js

```diff
@@ -61,20 +65,29 @@ helper.secrets.mockSuite(testing.suiteName(), ['google'], function(mock, skippin
     });
   });
 
-  const makeObject = async ({ name, data }) => {
-    const projectId = 'test-proj';
+  const projectId = 'test-proj';
+
+  const makeObject = async ({ name, data, hashes, gzipped }) => {
     const expires = taskcluster.fromNow('1 hour');
     const uploadId = taskcluster.slugid();
 
-    await helper.db.fns.create_object_for_upload(name, projectId, 'google', uploadId, expires, {}, expires);
+    await helper.db.fns.create_object_for_upload(name, projectId, 'aws', uploadId, expires, {}, expires);
```

Comment:
Should it really be `'aws'` here in a `google` test?

Reply:
It should not! :)

Reply:
Oh, sorry -- it should.  The "google" support is implemented through GCS's S3 emulation.